src/commands/importFromRegistry.ts

```diff
@@ -0,0 +1,168 @@
+import * as vscode from "vscode";
+import { Keychain } from "../keychain";
+
+export async function importFromRegistry(scope?: vscode.ConfigurationScope) {
+  const cmd = require("node-cmd");
+  const vsWinReg = require("vscode-windows-registry");
+  const hkeyLocalMachine = "HKEY_LOCAL_MACHINE";
+
+  const config = vscode.workspace.getConfiguration("intersystems", scope);
+  const serverDefinitions: any = config.get("servers");
+
+  const newServerNames = new Array<string>();
+  const serversMissingUsernames = new Array<string>();
+  const cancellationToken = new vscode.CancellationTokenSource().token;
+  cancellationToken.onCancellationRequested(() => {
+    vscode.window.showInformationMessage("Cancelled server import.");
+  });
+
+  await cmd.get(
+    "reg query " +
+      hkeyLocalMachine +
+      "\\SOFTWARE\\Intersystems\\Cache\\Servers",
```

Comment:
Good that the VSIX build now happens on Windows. However when I installed it and ran the command from the palette nothing seemed to happen.

On my 64-bit Windows 10 the actual registry key holding the server definitions is `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\InterSystems\Cache\Servers`

I am using 64-bit Node:
```
PS C:\Users\JohnM> node -p "process.arch"
x64
PS C:\Users\JohnM>
```

As an experiment I used regedit to export one of my server definitions into a file, then removed the WOW6432Node part of the path in the file, then imported the file. Now the command found that server.